### DISCUSSION TITLE
Advantages of appwrite.io

### DISCUSSION THREAD

**Discussion by davidjeba** [appwrite/appwrite#2520] [General]
Im building a CMS on Strapi.io which has extensions and so much features
just came across appwrite.io
is there any reason why appwrite is built from the scratch?
any advantages in scalability or something?

**Comment by eldadfux** [comment]
While Strapi is a CMS that was designed for content management, Appwrite is a backend-as-a-service or in short a BaaS that is designed for developers. Appwrite was designed from scratch mainly in order to have full control over the entire dev-experience. From the docs, to the installation, the integration or how the APIs are built. We spend tons of time choosing the naming conventions and how well the different components and services play with each other. By "owning" the stack we're able to introduce a better end-to-end experience and much more flexibility for developers using Appwrite.

**Reply by davidjeba** [reply]
Gastby has DEPLOY TO CLOUDFLARE script
https://www.gatsbyjs.com/docs/deploying-to-cloudflare-workers/

Can we have something like this ?
1.Deploy to Cloudflare
2.Deploy to Google App Engine
etc?

**Reply by eldadfux** [reply]
We can definitely discuss this, but a new, dedicated discussion might be better for it.
